### PR TITLE
bumped webpa-common, chrysom versions

### PR DESCRIPTION
**Pull request by kristinapathak**
also had to bump bascule and candlelight along the way; had to fix
breaking changes for that.

### CODE REVIEW

**Review by j-mai — APPROVED** [review]
The consul metrics update looks good to me!

File: main.go [hunk 1]
```diff
@@ -76,7 +74,7 @@ func scytale(arguments []string) int {
 		f = pflag.NewFlagSet(applicationName, pflag.ContinueOnError)
 		v = viper.New()
 
-		logger, metricsRegistry, webPA, err = server.Initialize(applicationName, arguments, f, v, webhook.Metrics, aws.Metrics, basculechecks.Metrics, basculemetrics.Metrics, chrysom.Metrics, Metrics, service.Metrics)
+		logger, metricsRegistry, webPA, err = server.Initialize(applicationName, arguments, f, v, webhook.Metrics, aws.Metrics, basculechecks.Metrics, basculemetrics.Metrics, consul.Metrics, Metrics, service.Metrics)
```
Comment (joe94) [inline main.go:77]: did we mean to add consul metrics in here? just double-checking as it wasn't there before

Comment (kristinapathak) [reply main.go:77]: Yup! Previously, the chrysom package declared metrics in the old webpa-common/xmetrics style.  Now that chrysom doesn't supply old style metrics, consul has to declare the metrics itself for the old way of registering metrics.

Comment (joe94) [reply main.go:77]: Just saw that, great!

**Review by joe94 — APPROVED** [review]

**Pull request merged**